### ISSUE TITLE
When training fullplace

### ISSUE THREAD

**Issue by PKUTAN** [Thinklab-SJTU/EDA-AI#8]
when I train, it do not seem to coverge
![image](https://user-images.githubusercontent.com/101382589/164147716-b08e50fa-ffd8-44f9-8feb-517ec1bed2ed.png)

**Comment by Rogerity** [comment]
Due to the immense action space, placement is actually a difficult RL learning task, so it is common to face such situation. There are several possible reasons for your curve, such as lack of pre-training or overfitting. To make training more stable, tuning hyperparameters such as reward funtion and learning rate may be helpful as well.

**Comment by PKUTAN** [comment]
Can you show the hyperparameters to train fullplace environment?
